### PR TITLE
Add support for Fritz!DECT lightbulbs

### PR DESCRIPTION
**Pull request by carpodaster**
So far, only switching them on and off support

Future refactoring could extract the `#toggle!` and `#active?` into a mixin like `OnOffSwitchable` (terrible name, but you get the idea :wink: ) to provide a uniform interface for all things on-off'able.

### CODE REVIEW

File: spec/fritzbox/smarthome/actor_spec.rb [hunk 1]
```diff
@@ -71,6 +71,37 @@
...
+    end
+  end
+
+  describe '.new_from_api' do
+    subject(:actor) { described_class.new_from_api(data) }
+
+    let(:data) { Hash.new }
+
+    context 'when setting the manufacturer' do
+      let!(:data) { { '@manufacturer' => 'Orang-Utan Klaus' } }
```
Comment (carpodaster) [inline spec/fritzbox/smarthome/actor_spec.rb:92]: @klausmeyer oh, just realized your first name. This is by no means a mockery, but rather a Helge Schneider reference. Happy to change this to something else

Comment (klausmeyer) [reply spec/fritzbox/smarthome/actor_spec.rb:92]: No worries. I had to lookup the reference but then I got it :grin:

**Review by klausmeyer — APPROVED** [review]
Thank you!

**Comment by klausmeyer** [comment]
Pushed a new release:

https://rubygems.org/gems/fritzbox-smarthome/versions/0.4.0

**Pull request merged**